packages/types/src/factors.ts

```diff
@@ -98,3 +108,8 @@ export type BackupCodeAttempt = {
   strategy: BackupCodeStrategy;
   code: string;
 };
+
+export type ResetPasswordCodeAttempt = {
```

Comment:
❓ i cannot find a reference in the PR. Is this used somewhere else?

Reply:
It is being used by this [PR](https://github.com/clerkinc/javascript/pull/1079/files)